src/patternfly/components/Form/form.scss

```diff
@@ -29,10 +29,10 @@ $pf-v6-c-form--m-horizontal--breakpoint-map: build-breakpoint-map("sm", "md", "l
   --#{$form}--GridGap: var(--pf-t--global--spacer--gap--group-to-group--vertical);
 
   // Group
-  --#{$form}__group--Gap: var(--pf-t--global--spacer--gap--group--horizontal);
+  --#{$form}__group--Gap: var(--pf-t--global--spacer--gap--group--vertical);
```

Comment:
Not sure why this was updated to `--horizontal`. The default layout for `__group` is a stack/vertical layout. Just reverting to what was there when we left some feedback around this block - https://github.com/patternfly/patternfly/pull/6957#discussion_r1747670189